### ISSUE TITLE
Compilation failure with msvc-14.0

### ISSUE THREAD

**Issue by pdimov** [boostorg/fiber#264]
```
compile-c-c++ bin.v2\libs\fiber\build\msvc-14.0\release\address-model-64\link-static\threading-multi\waker.obj
waker.cpp
.\boost/fiber/detail/spinlock_ttas_adaptive.hpp(48): error C2039: 'min': is not a member of 'std'
```

**Comment by olk** [comment]
Could you fetch the commit and test it again, please.

**Comment by pdimov** [comment]
This commit fixes the link=static build under msvc-14.0 (link=shared is still affected by the Intrusive issue.)

Don't merge it to master yet though, because master is frozen; wait for @mclow 's permission.

**Comment by olk** [comment]
ty
